ci/gpu/build.sh

```diff
@@ -81,14 +81,29 @@ logger "Adding ${CONDA_PREFIX}/lib to LD_LIBRARY_PATH"
 export LD_LIBRARY_PATH_CACHED=$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$CONDA_PREFIX/lib:$LD_LIBRARY_PATH
 
-logger "Build libcuml..."
+logger "Build libcuml, cuml, prims and bench targets..."
 $WORKSPACE/build.sh clean libcuml cuml prims bench -v
 
 logger "Resetting LD_LIBRARY_PATH..."
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH_CACHED
 export LD_LIBRARY_PATH_CACHED=""
 
+logger "Build treelite for GPU testing..."
+# Buildint treelite Python for testing is temporary while there is a pip/conda
+# treelite package
+
+cd $WORKSPACE/cpp/build/treelite/src/treelite
+mkdir build
```

Comment:
Will this cause issues if you run it twice so build exists? Should it be mkdir -p or something to ignore that?

Reply:
build directory doesn't exist in the GPU CI path due to the way cmake git cloning and building works, so it's not needed but can add it